homeassistant/components/generic/config_flow.py

```diff
@@ -261,6 +270,19 @@ async def async_test_stream(
     return {}
 
 
+def register_preview(hass: HomeAssistant):
+    """Set up previews for camera feeds during config flow."""
+    if not hass.data.get(DOMAIN):
+        hass.data[DOMAIN] = {}
```

Comment:
I learned from others this as the preferred way to do this:
```suggestion
    hass.data.setdefault(DOMAIN, {})
```